src/calculator.js

```diff
@@ -0,0 +1,32 @@
+import './calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="display"><span>0</span></div>
+      <div className="buttons">
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
+        <button type="button">%</button>
+        <button type="button" className="yellow">/</button>
+        <button type="button">7</button>
+        <button type="button">8</button>
+        <button type="button">9</button>
+        <button type="button" className="yellow">X</button>
+        <button type="button">4</button>
+        <button type="button">5</button>
+        <button type="button">6</button>
+        <button type="button" className="yellow">-</button>
+        <button type="button">1</button>
+        <button type="button">2</button>
+        <button type="button">3</button>
```

Comment:
- [ ] The `Calculator.js` component should have at least 1 child component. 🤔 This could have been a separate component (`Buttons`) in a separate file and rendered in the `Calculator` component. Smaller components are easy to fix, reusable, and make the app more scalable. :pray: 👍🏻